languages/java11/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,35 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+import java.util.HashMap;
+
+
+// This class defines your algorithm.
+class Algorithm extends AbstractAlgorithm<String, String>{
+
+    // This class member can be defined as anything you wish, and stores loaded state and mutable state between requests.
+     HashMap<String, String> context = new HashMap<>();
+
+    // This apply function defines the primary motive driver of your algorithm. Please ensure that the types defined in
```

Comment:
This language isn't very natural to me. "motive driver". Also "defned in class implementation above" almost made me think of the `HashMap` type that is immediately above this. Can we simplify this language?

As a strawman proposal:

```
// This apply function is the entry point for each API request. 
// The input argument and return types must match the types defined when extending the generic `AbstractAlgorithm`